tests/test_triples_factory.py

```diff
@@ -531,10 +531,10 @@ def assert_binary_io(self, tf, tf_cls):
         """Check the triples factory can be written and reloaded properly."""
         self.assertIsInstance(tf, tf_cls)
         with tempfile.TemporaryDirectory() as directory:
-            path = Path(directory) / "training.pt"
-            self.assertFalse(path.is_file())
+            path = Path(directory)
+            self.assertFalse(any(p.is_file() for p in path.iterdir()))
```

Comment:
not sure if these update tests make sense

Reply:
removed in e9686a2

I think it should be enough to check for correct loading of saved triples factory - in the end, we do not really care about the exact format on disk, but rather that we can properly load them again.

Reply:
Agreed! These should be "property-based" tests